vars/runIntegTestScript.groovy

```diff
@@ -0,0 +1,17 @@
+void call(Map args = [:]) {
+    String jobName = args.jobName ?: 'distribution-build-opensearch'
+    lib = library(identifier: 'jenkins@20211123', retriever: legacySCM(scm))
+    echo "args.buildManifest: ${args.buildManifest}"
+    echo "args.testManifest: ${args.testManifest}"
+    def buildManifest = lib.jenkins.BuildManifest.new(readYaml(file: args.buildManifest))
+    String artifactRootUrl = buildManifest.getArtifactRootUrl("${jobName}", "${BUILD_ID}")
+    echo "Artifact root URL: ${artifactRootUrl}"
+    
+    sh([
+        './test.sh',
+        'integ-test',
+        "${args.testManifest}",
+        "--paths opensearch=${artifactRootUrl}",
+        "--test-run-id ${BUILD_NUMBER}"
```

Comment:
This test run id can be omitted, this is a part of the legacy design that I don't believe we need to support.

Reply:
Removed in https://github.com/opensearch-project/opensearch-build/commit/e13948505ded132571d25c0f805ae499f9dfaf46